standard.md

```diff
@@ -4,19 +4,19 @@
-| **9c) Protection from harassment**                | If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+Indicator | Requirement
+--- | ---
+**1. Relevance to Sustainable Development Goals** | All projects must indicate the [Sustainable Development Goals](https://sdgs.un.org/goals) (SDGs) they are relevant to, and provide supporting links/documentation to support their relevance.
+**2. Use of Approved Open Licenses** | Projects must demonstrate the use of an approved open license. For open source software, only [OSI approved licenses](https://opensource.org/licenses) are accepted. For open content the use of a [Creative Commons license](https://creativecommons.org/licenses/) is required. While we encourage projects to use a license that allows for both derivatives and commercial reuse ([CC-BY](https://creativecommons.org/licenses/by/4.0/) and [CC-BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)), or dedicate content to the public domain ([CC0](https://creativecommons.org/choose/zero/)); licenses that do not allow for commercial reuse ([CC-BY-NC](https://creativecommons.org/licenses/by-nc/4.0/) and [CC-BY-NC-SA](https://creativecommons.org/licenses/by-nc-sa/4.0/)) are also accepted. For open data, an [Open Data Commons approved license](https://opendefinition.org/licenses/) is required. [The full license list is here for reference](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md).
```

Comment:
To hyperlink this whole sentence `The full license list is here for reference` does not look quite right. Could you propose alternative wording, perhaps like:

See [the full license list](https://github.com/unicef/publicgoods-candidates/blob/master/docs/licenses.md).

Reply:
Change made!